### DISCUSSION TITLE
Packaging pypandoc

### DISCUSSION THREAD

**Discussion by marph91** [pyinstaller/pyinstaller#8387] [Help] [answered]
Hi, I would like to package `pypandoc` with my application. Locally, I have installed `pypandoc_binary` ([via pip](https://github.com/JessicaTegner/pypandoc?tab=readme-ov-file#installing-via-pip)), which includes the pandoc binary (at `pypandoc.get_pandoc_path()`). But the binary doesn't seem to be included in the final pyinstaller-generated file, because there is always

```
OSError: No pandoc was found: either install pandoc and add it
to your PATH or or call pypandoc.download_pandoc(...) or
install pypandoc wheels with included pandoc.
```

I can mitigate this by specifying the binary manually (and handling it in the main application) `--add-binary="/path/to/anaconda/envs/py310/lib/python3.10/site-packages/pypandoc/files/pandoc:."`. But I would like to have it more generic, since I want to build it cross-platform later and the pandoc binary will be at different places.

Is there a best practice how to do this?

**Comment by rokm** [comment]
You can use [`--collect-data pypandoc`](https://pyinstaller.org/en/stable/usage.html#cmdoption-collect-data) (despite the name, it should pick up the executable).

**Comment by marph91** [comment, marked as answer]
Thanks! Actually, `--collect-data pypandoc` works for me. Might be a different pyinstaller version.

**Reply by rokm** [reply]
Yeah, I mistyped the name, sorry!
